engine/engine.go

```diff
@@ -525,6 +525,6 @@ func (e *Engine) Stop() {
 	e.ServicesWG.Wait()
 	err := gctlog.CloseLogger()
 	if err != nil {
-		log.Printf("Failed to close logger. Error: %v\n", err)
+		log.Printf("Failed to close log. Error: %v\n", err)
```

Comment:
logger

Reply:
:D vscode doesn't have as smart refactoring as Goland thought i picked up all these changes !

reverted this and the other instance

Reply:
No problemo, that's why we review things 😉